lib/foreman_maintain/utils/service/abstract.rb

```diff
@@ -31,7 +31,11 @@ def inspect
 
       def matches?(service)
         if service.is_a? String
-          service == @name
+          if service.include?('@*')
+            @name.include?(service.split('@*')[0])
```

Comment:
this is a systemd-specific feature, so shouldn't this be in `lib/foreman_maintain/utils/service/systemd.rb` instead?

(and then with a nice test added? :D)